core/src/main/java/io/neow3j/protocol/Neow3j.java

```diff
@@ -28,8 +29,8 @@ protected Neow3j(Neow3jConfig config) {
      * @param neow3jService a neow3j service instance, i.e., HTTP or IPC.
      * @return the new Neow3j instance.
      */
-    public static Neow3j build(Neow3jService neow3jService) {
-        return new JsonRpc2_0Neow3j(neow3jService, new Neow3jConfig());
+    public static Neow3j build(Neow3jService neow3jService) throws IOException {
```

Comment:
Why does the `build()` return now an `IOException` class?

Well, I know that somewhere else you're throwing it. Just want to know why this is a good decision, architecturally speaking.

Reply:
> Why does the `build()` return now an `IOException` class?

It still _returns_ a `Neow3j ` instance. It only _throws 
_ an `IOException`.

It does so because there's now an RPC happening during the creation of the returned Neow3j instance.

Reply:
I'm afraid that this would have a huge impact on tests and a huge refactoring of the code. Let's see.

What about throwing `Exception` instead?

Reply:
I've created a new custom runtime exception `Neow3jBuildException` for this now.